MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -277,6 +345,13 @@ void EnggDiffractionPresenter::processFocusTexture() {
 
   } else if (focusMode == 1) {
```

Comment:
For readability I'd suggest using an enum rather than 0/1 for the focus mode. That can be done while working on other issues/refactoring.